exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/RuntimeId.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.core.runtime;
+
+/**
+ * Represents a well-known runtime ids, as assigned by the Exonum core.
```

Comment:
Why `well-known`?

Reply:
Because the core hard-codes these ids in its source, they are not configurable.